### PR TITLE
CC-14555 Discount valid from/to supports time definition IG

### PR DESCRIPTION
**Pull request by AsonUnique**
Jira ticket: https://spryker.atlassian.net/browse/CC-14555

### CODE REVIEW

**Review by DmitryLymarenko — COMMENTED** [review]
there are js/css/translations added at https://github.com/spryker/spryker/pull/8914/files


please mention them as well

File: docs/scos/dev/feature-integration-guides/202200.0/promotions-and-discounts-feature-integration.md [hunk 1]
```diff
@@ -85,6 +85,16 @@ Make sure that the following changes have been triggered in transfer objects:
 | CalculatedDiscount.priority  | property | created | src/Generated/Shared/Transfer/CalculatedDiscountTransfer           |
 | DiscountGeneral.priority     | property | created | src/Generated/Shared/Transfer/DiscountGeneral                      |
 
+
+### 4) Set up behavior
```
Comment (DmitryLymarenko) [inline docs/scos/dev/feature-integration-guides/202200.0/promotions-and-discounts-feature-integration.md:89]: It looks more like `Build Zed UI frontend` than `behavior`

File: docs/scos/dev/feature-integration-guides/202200.0/promotions-and-discounts-feature-integration.md [hunk 2]
```diff
@@ -105,6 +115,8 @@ Make sure that discounts are calculated according to their priorities:
 
 {% info_block warningBox "Verification" %}
 
+Make sure that you can see a datetime picker dropdown when you click on **Valid From** and **Valid To** discount form fields.
```
Comment (DmitryLymarenko) [inline docs/scos/dev/feature-integration-guides/202200.0/promotions-and-discounts-feature-integration.md:118]: It's probably worth to specify details (where to find it, on which page)

**Review by DmitryLymarenko — APPROVED** [review]

File: docs/scos/dev/feature-integration-guides/202200.0/promotions-and-discounts-feature-integration.md [hunk 3]
```diff
@@ -75,49 +75,69 @@ Make sure that the following changes have been applied by checking your database
...
 {% info_block warningBox "Verification" %}
 
-Make sure that all labels and help tooltips in discount form has English and German translation.
+Make sure that you can submit `Discount Create` and `Discount Update` forms with specified date and time for **Valid From** and **Valid To** discount form fields:
 
-{% endinfo_block %}
+1. In the Back Office, go to **Merchandising** > **Discount**.
+2. Create new discount or update existing one, check that you can see `Discount` form.
+3. Make sure that you can see a datetime picker dropdown when you click on **Valid From** and **Valid To** discount form fields.
+4. Make sure that **Valid From** and **Valid To** discount form fields can accept selected date and time.
```
Comment (JacquieRooney) [inline docs/scos/dev/feature-integration-guides/202200.0/promotions-and-discounts-feature-integration.md:137]:
```suggestion
4. Make sure that **Valid From** and **Valid To** discount form fields can accept selected date and time, using keystroke entry for times not falling on the hour.
```

**Review by JacquieRooney — CHANGES_REQUESTED** [review]
Just some small adjustments to wording.

**Review by JacquieRooney — DISMISSED** [review]

File: docs/scos/dev/feature-integration-guides/202200.0/promotions-and-discounts-feature-integration.md [hunk 4]
```diff
@@ -75,49 +70,65 @@ Make sure that the following changes have been applied by checking your database
...
 
 {% endinfo_block %}
 
+{% info_block warningBox "Verification" %}
+
+Make sure that you can submit the *Discount Create* and *Discount Update* forms with specified date and time for **Valid From** and **Valid To** discount form fields:
+1. In the Back Office, go to **Merchandising** > **Discount**.
+2. Create a new discount or update an existing one, check that you can see the *Discount* form.
+3. Click on **Valid From** and **Valid To** discount form fields to ensure you can see the datetime drop-down menu.
+4. Make sure **Valid From** and **Valid To** discount form fields can accept the selected date and time, using keystroke entry for times not falling on the hour.
```
Comment (VadymSachenko) [inline docs/scos/dev/feature-integration-guides/202200.0/promotions-and-discounts-feature-integration.md:128]: What do you mean by "using keystroke entry for times not falling on the hour"?

File: docs/scos/dev/feature-integration-guides/202200.0/promotions-and-discounts-feature-integration.md [hunk 5]
```diff
@@ -75,49 +70,65 @@ Make sure that the following changes have been applied by checking your database
...
 
 {% endinfo_block %}
 
+{% info_block warningBox "Verification" %}
+
+Make sure that you can submit the *Discount Create* and *Discount Update* forms with specified date and time for **Valid From** and **Valid To** discount form fields:
+1. In the Back Office, go to **Merchandising** > **Discount**.
+2. Create a new discount or update an existing one, check that you can see the *Discount* form.
+3. Click on **Valid From** and **Valid To** fields to ensure you can see the calendar, where you can select specific date and time.
+4. Make sure **Valid From** and **Valid To** fields can accept the selected date and time, using keystroke entry for times not falling on the hour.
```
Comment (JacquieRooney) [inline docs/scos/dev/feature-integration-guides/202200.0/promotions-and-discounts-feature-integration.md:128]: With the date/time picker, you can only click on a time that is on the hour, for example 13:00. If you want to specify a time like 13:45, you would need to type it in. I wanted to make it clear that it was possible to select a time that wasn't exactly on the hour, hence this phrase. Feel free to adjust for clarity.

**Review by VadymSachenko — APPROVED** [review]
Approved from my side

**Pull request merged**